### ISSUE TITLE
How to achieve satisfactory results on test sets.

### ISSUE THREAD

**Issue by lh0616** [traveller59/second.pytorch#123]
@traveller59  Hello, this is a problem that has little to do with the code of SECOND , but I think you  have experience in this matter, so I hope to get your help.
I have constructed a 3D detection method, and achieved about 77.60% AP in the medium mode of KITTI validation set, and about 75% AP in the hard mode. But when I submitted to the KITTI test set, I only got the following results.
![image](https://user-images.githubusercontent.com/39039745/55271026-8ab8d900-52e1-11e9-8eb4-c3fafc683f6d.png)
As you know, we usually divide 7480 samples in the training set into two parts, half for training and the other half for evaluating the performance of the model. So I would like to consult you whether the model for test set should be one that is only trained on half of the training set, or one that needs to be trained on all 7480 samples in the training set.

**Comment by johleh** [comment]
All models suffer a performance drop on the test set compared to the validation results.
Training on more data, eg a custom split with 85/15 or 90/10 should increase test performance, but recently most models use the 50/50 split for test submissions to test under similar conditions.
